README.md

```diff
@@ -1,65 +1,73 @@
+**PHP components**. Symfony is used by thousands of web applications (including
+BlaBlaCar.com and Spotify.com) and most of the [popular PHP projects][2] (including
+Drupal and Magento).
 
 Installation
 ------------
 
-The best way to install Symfony is to use the [official Symfony Installer][7].
-It allows you to start a new project based on the version you want.
+* [Install Symfony][4] with Composer or with our own installer (see
+  [requirements details][3]).
+* Symfony follows the [semantic versioning][5] strictly, publishes "Long Term
+Support" (LTS) versions and has a [release process][6] that is predictable and
+business-friendly.
 
 Documentation
 -------------
 
-The "[Quick Tour][1]" tutorial gives you a first feeling of the framework. If,
-like us, you think that Symfony can help speed up your development and take
-the quality of your work to the next level, read the official
-[Symfony documentation][2].
+* Read the [Getting Started guide][7] if you are new to Symfony.
+* Master Symfony with the [Guides and Tutorials][8], the [Components docs][9]
+  and the [Best Practices][10] reference.
 
-Contributing
-------------
-
-Symfony is an open source, community-driven project. If you'd like to contribute,
-please read the [Contributing Code][3] part of the documentation. If you're submitting
-a pull request, please follow the guidelines in the [Submitting a Patch][4] section
-and use [Pull Request Template][5].
-
-Community Reviews
------------------
+Community
+---------
 
-If you don't feel ready to contribute code or patches, reviewing issues and pull 
-requests can be a great start to get involved and give back. In fact, people who 
-"triage" issues are the backbone to Symfony's success!  
-More information can be found in the [Community Reviews][8] guide.
+* [Join the Symfony Community][11] and meet other members at the [Symfony events][12].
+* [Get Symfony support][13] on StackOverflow, Slack, IRC, etc.
+* Follow us on [GitHub][14], [Twitter][15] and [Facebook][16].
 
-Running Symfony Tests
-----------------------
-
-Information on how to run the Symfony test suite can be found in the
-[Running Symfony Tests][6] section.
+Contributing
+------------
 
-[1]: https://symfony.com/doc/current/quick_tour/index.html
-[2]: https://symfony.com/doc/current/
-[3]: https://symfony.com/doc/current/contributing/code/index.html
-[4]: https://symfony.com/doc/current/contributing/code/patches.html#check-list
-[5]: https://symfony.com/doc/current/contributing/code/patches.html#make-a-pull-request
-[6]: https://symfony.com/doc/master/contributing/code/tests.html
-[7]: https://symfony.com/doc/current/book/installation.html#installing-the-symfony-installer
-[8]: https://symfony.com/doc/current/contributing/community/reviews.html
+Symfony is an Open Source, community-driven project. Join us [contributing code][17]
+or [contributing documentation][18] and you'll end up in the [Symfony contributors][19]
+Hall of Fame.
```

Comment:
The intention was not to be funny 😐  "Hall of Fame" is a commonly used name to refer to the place where you recognize the merits of some special people. See https://en.wikipedia.org/wiki/List_of_halls_and_walks_of_fame